conf/.env.production.sample

```diff
@@ -254,31 +254,31 @@ LDAP_TLS_NO_VERIFY=true
 # ---------------
  
 # Maximum allowed character count
-MAX_TOOT_CHARS=__MAX_TOOT_CHARS__
+MAX_TOOT_CHARS=500
 
 # Maximum number of pinned posts
-MAX_PINNED_TOOTS=__MAX_PINNED_TOOTS__
+MAX_PINNED_TOOTS=5
 
 # Maximum allowed bio characters
-MAX_BIO_CHARS=__MAX_BIO_CHARS__
+MAX_BIO_CHARS=500
 
 # Maximim number of profile fields allowed
-MAX_PROFILE_FIELDS=__MAX_PROFILE_FIELDS__
+MAX_PROFILE_FIELDS=4
 
 # Maximum allowed display name characters
-MAX_DISPLAY_NAME_CHARS=__MAX_DISPLAY_NAME_CHARS__
+MAX_DISPLAY_NAME_CHARS=30
 
 # Maximum allowed poll options
-MAX_POLL_OPTIONS=__MAX_POLL_OPTIONS__
+MAX_POLL_OPTIONS=5
 
 # Maximum allowed poll option characters
-MAX_POLL_OPTION_CHARS=__MAX_POLL_OPTION_CHARS__
+MAX_POLL_OPTION_CHARS=100
```

Comment:
Same. Couldn't have a little more space to breathe ? :)

Reply:
Those are the defaults and you're supposed to change it from the config panel (which is broken right now)

Reply:
But why stick to such strict defaults while we can offer more comfort *by default*, out-of-the-box ?
Especially if right now it's not easy to adjust this as its config panel is broken.